src/commands/run-circleci-bundle-update-pr.yml

```diff
@@ -31,14 +31,14 @@ parameters:
     default: "$CIRCLE_BRANCH"
     description: "Space separated branches. (e.g. 'master develop topic')"
   github_access_token:
-    type: string
-    default: "$GITHUB_ACCESS_TOKEN"
+    type: env_var_name
+    default: GITHUB_ACCESS_TOKEN
     description: |
       Your GitHub personal access token.
```

Comment:
Please update description and usage in https://github.com/sue445/circleci-ruby-orbs/blob/1.4.3/README.md#parameters-1.  

e.g. `Your GitHub personal access token key`, `Your GitHub personal access token env name` or any other good expression.